docs/team/linh.md

```diff
@@ -3,44 +3,43 @@ layout: page
 title: Linh's Project Portfolio Page
 ---
 
-## Project: AddressBook Level 3
+## Project: SunRez
 
-AddressBook - Level 3 is a desktop address book application used for teaching Software Engineering principles. The user interacts with it using a CLI, and it has a GUI created with JavaFX. It is written in Java, and has about 10 kLoC.
+SunRez is a desktop resident management application used for managing NUS hostels and residential colleges. The user interacts with it using a CLI, and it has a GUI created with JavaFX. It is written in Java, and has about 12 kLoC.
```

Comment:
Maybe expand the CLI and GUI into Command Line Interface (CLI) and similarly for GUI. Not sure if this affects the PPP too.